projects/pgai/pgai/vectorizer/loading.py

```diff
@@ -52,3 +52,11 @@ def load(self, row: dict[str, str]) -> LoadedDocument:
             file_path=row[self.column_name],
             file_type=guess_filetype(content, row[self.column_name]),
         )
+
+
+class DocumentLoadingError(Exception):
```

Comment:
WDYT about having a `LoadingError` base exception? In that way we can always raise an exception no matters the loader type.

Reply:
Sounds good, I'll switch to that.